.github/workflows/main.yml

```diff
@@ -0,0 +1,52 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build-nix:
+    name: Ubuntu xenial (x86_64) debug
+    runs-on: ubuntu-16.04
+    env:
+      BUILD_TYPE: Debug
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install dependencies
+        run: sudo apt-get update && sudo apt-get install -y libsdl2-dev libsdl2-image-dev libsdl2-mixer-dev zlib1g-dev
+      - name: Build for nix
+        run: bash -xe ci/build-nix.sh
+
+  build-nix-release:
+    name: Ubuntu xenial (x86_64) release
+    runs-on: ubuntu-16.04
```

Comment:
Xenial is going EOL in a few months, consider using bionic

Reply:
We're probably going to target `xenial` as long as it's not completely EOL with our AppImage builds, so I'd rather keep things consistent here.